### ISSUE TITLE
Boxart search keyword refinement

### ISSUE THREAD

**Issue by zeusz4u** [ClusterM/hakchi2#1185]
I've imported the whole NES library onto my NES Classic Mini, and sometimes the built-in search for covers retrieves different images than you expect, and sometimes you can't even find a cover that corresponds to the game.

It would be nice if you could manually refine the search terms for a certain game. I know it's doing probably a kind of "I'm feeling lucky" search for every title (like keywords "metroid nes cover"), but having the search term editable in the boxart browse window would definitely be an enhancement. For some games I could not find the right cover, had to open a web browser window and save the image, than load it afterwards in Hakchi... so much hassle as there were many games I couldn't find NES covers for.

Also not sure if anyone noticed, while scrolling the boxart results in Hakchi 2.21f, the windows jumps to the selected image while scrolling to the bottom and loading new images from Google. So once you scrolled down already do not want to see the first image again, and once you get to the bottom, it loads more images and in jumps to the first image again at the top.

**Comment by Shoothimnow** [comment]
Or, when it's done autosearching (as in... 'auto' searching), you can click and drag your own covers over the old to overwrite...

Also, Fature --> **Feature**

**Comment by zeusz4u** [comment]
Yeah, but for this you need to have your own covers, which are also saved from another google search from within your browser, then load them (either drag and drop or browse them in your local folders), so why not have this possibility to this right away from within hackchi to refine the search terms while clicking Google for that game if you're not satisfied by the results you see for a certain game, maybe the keywords are inappropriate...
